content/client-lib-development-guide/features.textile

```diff
@@ -561,23 +561,23 @@ h3(#realtime-presence). Presence
 ** @(RTP7a)@ Unsubscribe with no arguments unsubscribes the listener if previously subscribed with an action-specific subscription
 ** @(RTP7b)@ Unsubscribe with a single action argument unsubscribes the provided listener to all presence messages for that action
 * @(RTP8)@ @Presence#enter@ function:
-** @(RTP8a)@ Enters the current client into this channel, optionally with the data provided
+** @(RTP8a)@ Enters the current client into this channel, optionally with the data and/or extras provided
 ** @(RTP8b)@ Optionally a callback can be provided that is called for both success or failure to enter
 ** @(RTP8c)@ A @PRESENCE ProtocolMessage@ with a @PresenceMessage@ with the action @ENTER@ is sent to the Ably service. The @clientId@ attribute of the @PresenceMessage@ must not be present. Entering without an explicit @PresenceMessage#clientId@, implicitly uses the @clientId@ for the current connection
 ** @(RTP8d)@ Implicitly attaches the @Channel@ if the channel is in the @INITIALIZED@ state. However, if the channel is in the @DETACHED@ or @FAILED@ state, the @enter@ request results in an error
-** @(RTP8e)@ Optional data can be included when entering a channel that will be encoded / decoded as with normal messages. A test should exist to ensure data used with enter is encoded & decoded correctly. Also, when data is provided when entering, but no data is provided when leaving, the data attribute should be emitted in the @LEAVE@ event for this client
+** @(RTP8e)@ Optional data and/or extras can be included when entering a channel that will be encoded / decoded as with normal messages. A test should exist to ensure data and extras used with enter are encoded & decoded correctly. Also, when data and/or extras is provided when entering, but neither data nor extras are provided when leaving, the data attribute should be emitted in the @LEAVE@ event for this client
```

Comment:
From Slack:

> I have noticed that we oscillate between `word/word` and `word / word`

Where it makes sense, instead of a slash, lets have English: "... that will be encoded and decoded as with normal messages..."

Otherwise I think we don't need spaces around a slash.